config.toml

```diff
@@ -78,6 +78,8 @@ nextUrl = "https://kubernetes-io-vnext-staging.netlify.com/"
 githubWebsiteRepo = "github.com/kubernetes/website"
 githubWebsiteRaw = "raw.githubusercontent.com/kubernetes/website"
 
+survey-notice = true
```

Comment:
Try dropping the hyphen and adjusting _if_ statements accordingly. Hugo sometimes behaves oddly around punctuation in params.
> `surveynotice = true`

Reply:
That didn't make any difference. It's something with the way the docs/home page is constructed at compile time. I'm missing something fundamental. Is the person who put the Hugo site together still around?

Reply:
You need to drop the hyphen that then change the lines in the `layouts/shortcodes/survey-notice.html` and `layouts/partials/survey-notice.html` pages that look like this:

```html
{{ if .Param "survey-notice"}}
```

to this:

```html
{{ if .Param "surveynotice"}}
```